hyrax/app/models/solr_document.rb

```diff
@@ -93,4 +93,20 @@ def synthesis_and_processing
   def custom_property
     self[Solrizer.solr_name('custom_property', :displayable)]
   end
+
+  def complex_event
+    self[Solrizer.solr_name('complex_event', :displayable)]
+  end
+
+  def issue
+    self[Solrizer.solr_name('issue', :displayable)]
+  end
+
+  def place
+    self[Solrizer.solr_name('place', :stored_searchable)]
+  end
```

Comment:
need to replace with :displayable with :stored_searchable based on [model line 70](https://github.com/antleaf/nims-hyrax/blob/develop/hyrax/app/models/publication.rb#L70)

Reply:
changed